Add exclusive fullscreen checkbox to advanced video options

FrozenFish24 opened the pull request:
`-window-mode exclusive` arg wasn't working, so I added it as a proper option.

![exclusive](https://user-images.githubusercontent.com/12547453/63651649-9eca0f00-c789-11e9-9e77-1a536016edaa.png)

TheLacus commented:
Could you _fixup_ the second commit into the first?

Interkarma commented:
Hey there, thanks for the PR! I'm happy to have an exclusive fullscreen option. If nothing else it will help people using gsync/freesync.

One thing, could you please change the option from "Exclusive" to "ExclusiveFullscreen"? This makes it a bit clearer from the setting along what it's for. When existing users sync this setting for the first time, it will appear at the bottom of the group list and won't be as apparent that it's related to fullscreen.

I'll test changes when I can and let you know if any feedback.

FrozenFish24 commented:
Renamed to `ExclusiveFullscreen` and squashed commits. However, I think I've run into a Unity bug.

Switching between Borderless Fullscreen and Exclusive Fullscreen seems to ignore the resolution and forces native res until you `Screen.SetResolution` a second time. For example if I set my res to 1280x720, check `Fullscreen` and hit apply then check `Exclusive Fullscreen` and apply, the res changes to 1920x1080 (my native) despite appearing correct in the options and ini.

Starting the game does another `Screen.SetResolution` so maybe this isn't such a problem.

I think this is the same issue:
https://issuetracker.unity3d.com/issues/screen-dot-setresolution-width-height-fullscreenmode-disregards-resolution-when-changing-to-exclusivefullscreen

FrozenFish24 commented:
Good thinking, I've commented out the UI option.

Interkarma commented on Assets/Scripts/Game/UserInterfaceWindows/DaggerfallPauseOptionsWindow.cs:
```diff
@@ -293,6 +293,15 @@ private void ControlsButton_OnMouseClick(BaseScreenComponent sender, Vector2 pos
         {
             fullScreenTick.Enabled = DaggerfallUnity.Settings.Fullscreen = Screen.fullScreen = !Screen.fullScreen;
+
+            if (DaggerfallUnity.Settings.Fullscreen && DaggerfallUnity.Settings.ExclusiveFullscreen)
```
Just FYI, I'll merge this as-is but the Fullscreen button in DF will soon be reverted to toggle large HUD on and off. It had a different meaning in classic than the modern fullscreen/windowed.

Interkarma commented:
Thank you, merging this one now!
